tests/phpunit/tests/html-api/wpHtmlProcessorMetaTag.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Processor META tag handling.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ *
+ * @since TBD
```

Comment:
let’s put this in 6.9, though as test files, should we not put the `@ticket` in instead of a `@since` version?

Reply:
I'm not sure here, ticket seems overly specific for an entire test suite. I think other test suites use since? I've always thought of tickets as specific to individual tests.

```suggestion
 * @since 6.9
```